src/Components/SectionTitle/SectionTitle.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './SectionTitle.css';
+
+function SectionTitle(props) {
+    return (
+        <h2>{props.text}</h2>
+    )
+}
```

Comment:
It's really unnecessary to create components for all tags you use. Look, you've created directory with 3 files, forward props instead of using <h2>  into the parent component. Keep it simplier